src/pivotal-ui/components/links/links.scss

```diff
@@ -49,30 +48,32 @@ a{
   &:hover, &:focus {
     cursor: pointer;
     text-decoration: none;
+    color: lighten($blue-2, 5%);
+  }
+
+  &.active{
+    color: darken($blue-2, 5%);
   }
 
   &.link-text {
-    &:not(.link-lowlight):hover {
-      text-decoration: underline;
-    }
+    font-weight: $link-lowlight-font-weight;
   }
+
+
 }
 
 
+
 .link-lowlight {
   font-weight: $link-lowlight-font-weight;
   color: $link-lowlight-color;
 
   &:hover {
-    color: $link-lowlight-hover-color;
+    color: lighten($link-lowlight-color, 5%);
   }
-}
-
-.link-lowlight-alt {
-  color: $link-lowlight-alt-color;
 
-  &:hover {
```

Comment:
Maybe the default link hover color is what you also wanted for the lowlight link?